Assets/Scripts/Features/Gameplay/Camera/CameraDragging.cs

```diff
@@ -91,7 +102,7 @@ private void MoveObject()
     }
     private void Update()
```

Comment:
WHITE REGEL ** ANGRY EXHALE **  \(!!˚☐˚)/

Reply:
Your comment was not completely in English.

Reply:
I do not know wet jij aan het praten bent